mautrix_facebook/user.py

```diff
@@ -31,6 +31,22 @@
 from .db import (User as DBUser, UserPortal as DBUserPortal, Contact as DBContact, ThreadType,
                  Portal as DBPortal)
 from . import portal as po, puppet as pu
+from prometheus_client import Summary, Enum
+
+METRIC_SYNC_THREADS = Summary('bridge_sync_threads', 'calls to sync_threads')
+METRIC_RESYNC = Summary('bridge_on_resync', 'calls to on_resync')
+METRIC_UNKNOWN_EVENT = Summary('bridge_on_unknown_event', 'calls to on_unknown_event')
+METRIC_MEMBERS_ADDED = Summary('bridge_on_members_added', 'calls to on_members_added')
+METRIC_MEMBER_REMOVED = Summary('bridge_on_member_removed', 'calls to on_member_removed')
+METRIC_TYPING = Summary('bridge_on_typing', 'calls to on_typing')
+METRIC_PRESENCE = Summary('bridge_on_presence', 'calls to on_presence')
+METRIC_REACTION = Summary('bridge_on_reaction', 'calls to on_reaction')
+METRIC_MESSAGE_UNSENT = Summary('bridge_on_unsent', 'calls to on_unsent')
+METRIC_MESSAGE_SEEN = Summary('bridge_on_message_seen', 'calls to on_message_seen')
+METRIC_TITLE_CHANGE = Summary('bridge_on_title_change', 'calls to on_title_change')
+METRIC_MESSAGE = Summary('bridge_on_message', 'calls to on_message')
+METRIC_LOGGED_IN = Enum('bridge_logged_in', 'Bridge Logged in', states=['true', 'false'])
+METRIC_CONNECTED = Enum('bridge_connected', 'Bridge Connected', states=['true', 'false'])
```

Comment:
At least these two should probably include the user ID somewhere

Reply:
No need assuming the bridge is only ever used by one user - prometheus will collect additional metadata from wherever the data is collected - for example with Kubernetes, the namespace and labels of the user are collected with every stat. See https://grafana.novachat-helsinki.kubesail-ops.com/d/ktQTLJOGk/bridge-dashboard?orgId=1 for more :)

Reply:
The bridge supports multiple users, so that assumption isn't always true. Can the user ID be added in some way that's ignored or overridden in the kubernetes setup?